<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#ff0000;&quot;&gt;Tinh So Ngay Trong Thang&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Nam">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Thang">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nhap nam :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nhap thang : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ThucHien">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Thuc hien</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TiepTuc">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiep tuc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Thoat">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Thoat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ngay">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
